README.md

```diff
@@ -1,4 +1,5 @@
 # Core Vanilla JavaScript
+Grey Toucan Please see solo study Guide for my work.
 
 This repo contains resources, challenges and plans for learning the basics of Javascripts.
 
```

Comment:
I hope you learned a lot about Vanilla JS while doing the tutorials, not really a lot to review in the repo...